site/content/how-to-monetize-apps/monetization.md

```diff
@@ -0,0 +1,92 @@
+---
+weight: 10 
+---
+
+# How to monetize apps?
+
+<!--Abstract, short intro -->
+Whether your app is already up and running or you're just starting off, the topic of **app monetization** will eventually cross your mind. If you don't know how to go about generating income from your apps, this is the place for you to start.
+
+
+<!-- Intro for newcomers -->
+## Introduction
+If you're new to LiveChat, here's how things work in here. LiveChat can be extended with various applications built by developers. All of this is done to meet the specific business needs of those who use LiveChat. You, as a developer and app author, can sell it on our [Marketplace](https://www.livechatinc.com/marketplace/). 
+
+If you'd like to know what types of applications can be built upon LiveChat Platform, check out the [Building LiveChat apps](/building-apps-for-livechat/) doc. 
+
+The place to start creating your apps with different building blocks is our [Developer Console](https://developers.livechatinc.com/console/). If your app is ready, you can continue with the monetization process. 
+
+<!-- How the process looks -->
+## Core information
+
+Here's the core info about monetization at **LiveChat**:
+
+* You can perform all the actions in one place- [Developer Console](https://developers.livechatinc.com/console/), which speeds up the whole procedure. 
+* We use PayPal as our billing platform. 
+* The provision is 20%, so if you set the app price to be 10$, you will earn 8$ net. 
+* We offer two billing cycles: **once a month** and **one-time payment**. We recommend the recurring payment option. We're currently working on the implementation on **micropayments**, so stay tuned for this one!
+<!-- Czemu rekomendujemy recurring? Co mozna jeszcze powiedziec o micropayments?-->
+
+## Your role vs. Our role
+
+### **Billing**
+**Your part** in this process is to go through the **monetization procedure** and, most importantly, **decide on the app payment details**. 
```

Comment:
I'd highlight the low requirements on the developer side:
>Your part is to simply **pick the app billing model** and to set up the monetization. It's as simple as using the App Monetization Building Block. Payments will be collected automatically depending on the billing cycle you choose.

Reply:
You're right, good too emphasise that :) What about: **The only action on your side is to pick up the app billing model and set up monetization.**

Reply:
Perfect!